image.go

```diff
@@ -20,6 +20,10 @@ type Image interface {
 	Target() ocispec.Descriptor
 	// Unpack unpacks the image's content into a snapshot
 	Unpack(context.Context, string) error
+	// RootFS returns the image digests
+	RootFS(ctx context.Context) ([]digest.Digest, error)
+	// Size returns the image size
```

Comment:
@abhinandanpb Probably want to change the comments to https://github.com/containerd/containerd/blob/master/images/image.go#L46-L68.

`RootFS returns the image digests` and `Size returns the image size` seem confusing to me.

Sorry for the delay, saw the merge notification and came back comment. :p